README.md

```diff
@@ -71,7 +71,7 @@ rails console
 ```
 3. In the console, run the following command to invite a user:
 ```bash 
-User.invite!(email: <some_test_email>)
+User.invite!(email: <some_test_email>, first_name: <user_first_name>, last_name: <user_last_name></user_last_name>)
```

Comment:
I forgot that we can change this back to just say
```bash
User.invite!(email: <some_test_email>)
```
Because we've added the default name stuff to the `invite!` method.